egs/tedlium3/ASR/pruned_transducer_stateless/decoder.py

```diff
@@ -0,0 +1,105 @@
+# Copyright    2021  Xiaomi Corp.        (authors: Fangjun Kuang
```

Comment:
Can you use a symlink to the same file in the librispeech recipe to replace this file?
You can put the line `self.unk_id = unk_id` in that file.